ownership-chain/pallets/asset-metadata-extender/src/mock.rs

```diff
@@ -0,0 +1,69 @@
+use crate as pallet_asset_metadata_extender;
+use frame_support::{
+	parameter_types,
+	traits::{ConstU16, ConstU64},
+	weights::constants::RocksDbWeight,
+};
+use sp_core::{H160, H256};
+use sp_runtime::{
+	traits::{BlakeTwo256, IdentityLookup},
+	BoundedVec, BuildStorage,
+};
+
+type Block = frame_system::mocking::MockBlock<Test>;
+
+// Configure a mock runtime to test the pallet.
+frame_support::construct_runtime!(
+	pub enum Test
+	{
+		System: frame_system,
+		AssetMetadataExtender: pallet_asset_metadata_extender,
+	}
+);
+
+pub type AccountId = H160;
```

Comment:
don't need to be public : `AccountIdOf<...>` seems the pattern proposed by the entire pallet